dtos.py

```diff
@@ -0,0 +1,47 @@
+from dataclasses import dataclass, field
+from typing import Union
+
+from dataclasses_json import dataclass_json
+
+
+@dataclass_json
+@dataclass
+class CommentDto:
+    path: str = field(default="")
+    start_line: int = field(default=0)
+    end_line: int = field(default=0)
+    original_start_line: int = field(default=0)
+    original_end_line: int = field(default=0)
+    body: str = field(default="")
```

Comment:
why every field has a default value? are you expecting some data to not have these fields?